### ISSUE TITLE
Nammu.askForPermission fragment not available in 1.1.1

### ISSUE THREAD

**Issue by fouadkada** [tajchert/Nammu#17]
in version 1.1.1 the only method signatures for askForPermission are the following

```
askForPermission(Activity activity, String permission, PermissionCallback permissionCallback)
```
```
askForPermission(Activity activity, String[] permissions, PermissionCallback permissionCallback)
```

There is no askForPermission for Fragment as shown in the sample app

**Comment by tajchert** [comment]
This is very weird as it is in the code : https://github.com/tajchert/Nammu/blob/master/nammu/src/main/java/pl/tajchert/nammu/Nammu.java#L96
Will look for solution on this.

**Comment by tajchert** [comment]
Should be fixed and working well in 1.1.2 version.
https://jitpack.io/#tajchert/Nammu/1.1.2
